packages/cli/src/cli.ts

```diff
@@ -47,20 +47,32 @@ export function cli({
                         description:
                             'List of tables to be included in the generation',
                     })
-                    .array('tables');
+                    .array('tables')
+                    .config('configFile')
+                    .parserConfiguration({
+                        'dot-notation': false,
+                        'strip-aliased': true,
+                    });
             },
             async (argv) => {
                 console.warn('Connecting to database schema...');
 
-                const { schema } = await generateSchema({
+                const result = await generateSchema({
                     connectionString: argv.connectionString,
                     out: argv.out,
                     defaultSchema: argv.defaultSchema,
                     schemas: argv.schemas,
                     tables: argv.tables,
+                    schemaDefOverrides: argv.schemaDefOverrides,
                 });
-                const tables = Object.keys(schema.properties ?? {});
+
+                // TODO: We want to check if there were any overrides passed
```

Comment:
This should be implemented in this MR. It's a very simple check.

Just check if the generated tables include the overrides.

Reply:
Done now.. I added the comment so that I'll remember to add it this morning